cmd/agent/dist/conf.d/oracle-dbm.d/conf.yaml.example

```diff
@@ -15,103 +15,121 @@ instances:
+  # instant_client: false
+
+  ## @param username - string - required
+  ## Username for the Datadog-Oracle server check user. The user has to exist in CDB.
+  # 
+  # username: <USERNAME>
 
-    ## @param password - string - required
-    ## Password for the Datadog-Oracle check user.
-    #
-    # password: <PASSWORD>
+  ## @param password - string - required
+  ## Password for the Datadog-Oracle check user.
+  #
+  # password: <PASSWORD>
 
-    ## @param reported_hostname - string - optional
-    ## Set the reported hostname for this instance. This value overrides the hostname detected by the Agent
+  ## @param reported_hostname - string - optional
+  ## Set the reported hostname for this instance. This value overrides the hostname detected by the Agent
 
-    ## @param dbm - boolean - optional - default: false
-    ## Set to `true` to enable Database Monitoring.
-    #
-    # dbm: false
+  ## @param dbm - boolean - optional - default: false
+  ## Set to `true` to enable Database Monitoring.
+  #
+  # dbm: false
 
-    ## Configure how the SQL obfuscator behaves.
-    ## Note: This option only applies when `dbm` is enabled.
-    #
-    # obfuscator_options:
-
-        ## @param replace_digits - boolean - optional - default: false
-        ## Set to `true` to replace digits in identifiers and table names with question marks in your SQL statements.
-        ## Note: This option also applies to extracted tables using `collect_tables`.
-        #
-        # replace_digits: false
-
-        ## @param collect_metadata - boolean - optional - default: true
-        ## Set to `false` to disable the collection of metadata in your SQL statements.
-        ## Metadata includes things such as tables, commands, and comments.
-        #
-        # collect_metadata: true
-
-        ## @param collect_tables - boolean - optional - default: true
-        ## Set to `false` to disable the collection of tables in your SQL statements.
-        ## Requires `collect_metadata: true`.
-        #
-        # collect_tables: true
-
-        ## @param collect_commands - boolean - optional - default: true
-        ## Set to `false` to disable the collection of commands in your SQL statements.
-        ## Requires `collect_metadata: true`.
-        ##
-        ## Examples: SELECT, UPDATE, DELETE, etc.
-        #
-        # collect_commands: true
-
-        ## @param collect_comments - boolean - optional - default: true
-        ## Set to `false` to disable the collection of comments in your SQL statements.
-        ## Requires `collect_metadata: true`.
-        #
-        # collect_comments: true
-
-      ## @param tags - list of strings - optional
-      ## A list of tags to attach to every metric and service check emitted by this instance.
+  ## Configure collection of query metrics
+  #
+  # query_metrics: true
+
+      ## @param enabled - boolean - optional - default: true
+      ## Enable collection of query metrics. Requires `dbm: true`.
+      #
+      # enabled: true
```

Comment:
this looks incorrect when comparing to how you have [defined this struct](https://github.com/DataDog/datadog-agent/pull/17000/files#diff-365baa745bde9428c3cd5738c844f63061a48ccfa1848b48bfc6f4d20176bfdaR40)

should this be:
```
QueryMetrics map[string]QueryMetricConfig `yaml:"query_metrics"`
...
...

type QueryMetricConfig struct {
Enabled bool `yaml:"enabled"`
}
```

Reply:
Corrected now, thx. I also added the `query_samples` parameter to control sampling.

Reply:
> so if we want to be consistent across our integrations the config query_samples refers to execution plan collection and query_activity refers to activity/load sampling
> 
> Maybe it makes sense to do something different here with the new integration, like:
> 
> query_samples -> activity/load samples
> collect_execution_plans -> plan collections
> idk, what do you think?

Using `query_samples` for controlling execution plans goes agains my intuition. "Sample" is the front end annotation for query samples, so `query_samples` is better aligned with the GUI:

![image](https://github.com/DataDog/datadog-agent/assets/18366081/dc86ce72-9285-4d1f-8037-9e79690643fb)

Also @justiniso recognised the inconsistent naming and [wants to change it](https://dd.slack.com/archives/C027TGQFUMR/p1683224818003169?thread_ts=1682425251.404039&cid=C027TGQFUMR)

I have already introduced the parameter `execution_plans` for controlling the collection of execution plans. I could change it to `collect_execution_plans` if that's more appropriate. That parameter will be introduced in a [separate branch](https://github.com/DataDog/datadog-agent/blob/nenad.noveljic/oracle-execution-plans/pkg/collector/corechecks/oracle-dbm/config/config.go#L42) that implements only execution plan collection functionality.